rotkehlchen/globaldb/updates.py

```diff
@@ -203,47 +204,83 @@ def _parse_optional_int(self, value: str, name: str, insert_text: str) -> Option
 
     def _parse_asset_data(self, insert_text: str) -> ParsedAssetData:
         match = self.assets_re.match(insert_text)
+
```

Comment:
no need to add extra spaces here. The if is part of the same initialization of `match` here